ftl/core/deck-config.ftl

```diff
@@ -414,22 +413,22 @@ deck-config-reschedule-cards-warning =
 deck-config-ignore-before-tooltip = 
     If set, reviews before the provided date will be ignored when optimizing & evaluating FSRS parameters.
     This can be useful if you imported someone else's scheduling data, or have changed the way you use the answer buttons.
-deck-config-compute-optimal-weights-tooltip =
-    Once you've done 1000+ reviews in Anki, you can use the Optimize button to analyze your review history,
-    and automatically generate parameters that are optimal for your memory and the content you're studying.
-    If you have decks that vary wildly in difficulty, it is recommended to assign them separate presets, as
-    the parameters for easy decks and hard decks will be different. There is no need to optimize your parameters
-    frequently - once every few months is sufficient.
+deck-config-compute-optimal-weights-tooltip2 =
+    When you click the Optimize button, FSRS will analyze your review history to generate parameters that are 
+    optimal for your memory patterns and the content you're studying. If your decks vary wildly in difficulty, it 
```

Comment:
I'm not really fond of "memory patterns". How about something like the following?

```suggestion
    optimal for your brain, and the content you're studying. If your decks vary wildly in difficulty, it 
```

or even just

```suggestion
    optimal for the content you're studying. If your decks vary wildly in difficulty, it 
```

Reply:
If you want, we can just remove the word "patterns".

Reply:
`optimal for your memory` sounds fine to me.